Source/qol/stash.cpp

```diff
@@ -272,13 +241,14 @@ void InitStash()
 	StashNavButtonArt = LoadClx("data\\stashnavbtns.clx");
 }
 
-void TransferItemToInventory(Player &player, uint16_t itemId)
+void TransferItemToInventory(Player &player, uint16_t itemIndex)
 {
-	if (itemId == uint16_t(-1)) {
+	if (itemIndex == InvalidItemId) {
 		return;
 	}
 
-	Item &item = Stash.stashList[itemId];
+	Item &item = Stash.stashList[itemIndex];
+	Point originalPosition = item.position;

```

Comment:
Reading the position of an empty item isn't ideal. The declaration of `originalPosition` should move to immediately before the call to `AutoPlaceItemInInventory()`

Reply:
Suggestion applied in most recent push